### DISCUSSION TITLE
Can I use the sample code as a progression in my app?

### DISCUSSION THREAD

**Discussion by GidaeYeo** [creativecreatorormaybenot/funvas#208] [Q&A] [answered]
Thank you for creating such great code.
Do you mind if I use the following example code as progress in my app?
I will be mentioning this project on the About Page of my app.
https://github.com/creativecreatorormaybenot/funvas/blob/main/funvas_tweets/lib/src/32.dart

**Comment by creativecreatorormaybenot** [comment, marked as answer]
Hi - you may use the animation if you properly attribute in the about page of you app according to the project's open source license.

**Reply by Blacktaler** [reply]
Hi, I'm saying this if you didn't notice that, there's a copy of your package wakelock inside the packages of fluttercommunity. 

https://pub.dev/packages/wakelock_plus

and
this one also

https://pub.dev/packages/flu_wake_lock
